### ISSUE TITLE
Pixelation ScreenCapture in Android

### ISSUE THREAD

**Issue by mekya** [ant-media/Ant-Media-Server#4733]
### Short description
There is a similar pixelation issue(https://github.com/ant-media/Ant-Media-Server/issues/4382) while sending the stream with Android SDK.  

Steps 
- Publish ScreenCapture from Android 
- Play on the web browser
- Agresively swipe the view in Android
- Monitor the player in the web brower

Expected:
Smooth play and not pixelation

Actual:
Pixelation in Android

**Comment by shibupanda** [comment]
Yes @mekya this is current. Pixelation issue is happening from every action or navigation.
Quality also not up to the mark.
